src/Nodes/Texts/SVGText.php

```diff
@@ -1,13 +1,83 @@
 <?php
-
 namespace SVG\Nodes\Texts;
 
+use SVG\Nodes\Structures\SVGFont;
 use SVG\Nodes\SVGNodeContainer;
+use SVG\Rasterization\SVGRasterizer;
 
 /**
  * Represents the SVG tag 'text'.
+ *
+ * Usage:
+ *
+ * $svg = new \SVG\SVGImage(600, 400);
+ *
+ * $font = new \SVG\Nodes\Structures\SVGFont('openGost', 'OpenGostTypeA-Regular.ttf');
+ * $svg->getDocument()->addChild($font);
+ *
+ * $svg->getDocument()->addChild(
+ *   (new \SVG\Nodes\Texts\SVGText('hello', 50, 50))
+ *     ->setFont($font)
+ *     ->setSize(15)
+ *     ->setStyle('stroke', '#f00')
+ *     ->setStyle('stroke-width', 1)
+ * );
+ *
  */
 class SVGText extends SVGNodeContainer
 {
     const TAG_NAME = 'text';
+
+    /**
+     * @var SVGFont
+     */
+    private $font;
+
+    public function __construct($text = '', $x = 0, $y = 0)
+    {
+        parent::__construct();
+        $this->setValue($text);
+        if($x > 0) {
+            $this->setAttribute('x', $x);
+        }
+        if($y > 0) {
+            $this->setAttribute('y', $y);
+        }
```

Comment:
Why can text not start outside the viewport? I think that would be a perfectly valid thing, and these conditionals should not be here.